dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/CoreFeature.kt

```diff
@@ -676,6 +683,11 @@ internal class CoreFeature(
     // endregion
 
     companion object {
+        internal const val SDK_INITIALIZED_IN_SECONDARY_PROCESS_WARNING_MESSAGE =
+            "Datadog SDK was initialized in a secondary process. We strongly recommend to initialize the SDK" +
+                "in the application main process otherwise the SDK would not function properly. The only scenario" +
+                "where this is acceptable is where there will be 2 instances of the SDKs running " +
+                "in the same time and one instance is guaranteed to be in the main process."
```

Comment:
```suggestion
        internal const val SDK_INITIALIZED_IN_SECONDARY_PROCESS_WARNING_MESSAGE =
            "Datadog SDK was initialized in a secondary process. We strongly recommend to initialize the SDK" +
                " in the application main process otherwise the SDK would not function properly. The only scenario" +
                " where this is acceptable is where there will be 2 instances of the SDKs running" +
                " in the same time and one instance is guaranteed to be in the main process."
```

I feel though that this is very verbose. Maybe we can re-word it simpler? Like tell user that SDK _must_ have instance initialized in the main process and that is it. And speaking of `the SDK would not function properly`: maybe we can just say that no data will be recorded (well, to be precise - uploaded).

This warning though raises a question why such limitation exists and why cannot we solve it (in theory we could?).

Reply:
Yes I wanted to present all the possibilities in the message but I guess we can lower the amount of information. Regarding your second point there are some strong reasons why we took this approach. Let's debate about this.